pkg/modules/application_mobility.go

```diff
@@ -407,51 +458,79 @@ func getVelero(op utils.OperatorConfig, cr csmv1.ContainerStorageModule) (string
 }
 
 // IsAppMobilityComponentEnabled - check if Application Mobility componenets are enabled
-/*
-func IsVolumaeSnapshotEnabled(ctx context.Context, instance csmv1.ContainerStorageModule, r ReconcileCSM, mod csmv1.ModuleType, componentType string) bool {
-	appMobilityEnabled, appmobility := IsModuleEnabled(ctx, instance, mod)
-	if !appMobilityEnabled {
-		return false
-	}
+// getVelero - gets the velero-deployment manifest
+func getUseVolumeSnapshot(op utils.OperatorConfig, cr csmv1.ContainerStorageModule) (string, error) {
+	YamlString := ""
 
-	for _, c := range appmobility.Components {
-		if c.Name == componentType && *c.Enabled {
-			return true
-		}
+	appMob, err := getAppMobilityModule(cr)
+	if err != nil {
+		return YamlString, err
 	}
 
-	return false
-}
-
-// IsAppMobilityComponentEnabled - check if Application Mobility componenets are enabled
-func IsCleanupCRDEnabled(ctx context.Context, instance csmv1.ContainerStorageModule, r ReconcileCSM, mod csmv1.ModuleType, componentType string) bool {
-	appMobilityEnabled, appmobility := IsModuleEnabled(ctx, instance, mod)
-	if !appMobilityEnabled {
-		return false
+	VolSnapshotPath := fmt.Sprintf("%s/moduleconfig/application-mobility/%s/%s", op.ConfigDirectory, appMob.ConfigVersion, UseVolSnapshotManifest)
+	buf, err := os.ReadFile(filepath.Clean(VolSnapshotPath))
+	if err != nil {
+		return YamlString, err
 	}
 
-	for _, c := range appmobility.Components {
-		if c.Name == componentType && *c.Enabled {
-			return true
+	YamlString = string(buf)
+	VolSnapshotlocation_name := ""
+	Velero_NS := ""
+	Provider := ""
+	for _, component := range appMob.Components {
+		if component.Name == AppMobVeleroComponent {
+			for _, env := range component.Envs {
+				if strings.Contains(VolSnapshotlocation, env.Name) {
+					VolSnapshotlocation_name = env.Value
+				}
+				if strings.Contains(VeleroNamespace, env.Name) {
+					Velero_NS = env.Value
+				}
+				if strings.Contains(ConfigProvider, env.Name) {
+					Provider = env.Value
+				}
+			}
 		}
 	}
 
-	return false
+	YamlString = strings.ReplaceAll(YamlString, VeleroNamespace, Velero_NS)
+	YamlString = strings.ReplaceAll(YamlString, VolSnapshotlocation, VolSnapshotlocation_name)
+	YamlString = strings.ReplaceAll(YamlString, ConfigProvider, Provider)
+
+	return YamlString, nil
 }
 
-// IsAppMobilityComponentEnabled - check if Application Mobility componenets are enabled
-func IsdeployResticEnabled(ctx context.Context, instance csmv1.ContainerStorageModule, r ReconcileCSM, mod csmv1.ModuleType, componentType string) bool {
-	appMobilityEnabled, appmobility := IsModuleEnabled(ctx, instance, mod)
-	if !appMobilityEnabled {
-		return false
+func getCleanupcrds(op utils.OperatorConfig, cr csmv1.ContainerStorageModule) (string, error) {
+	YamlString := ""
```

Comment:
local variables (not being exported outside of the function) need to be camelcase with lowercase first letter

Reply:
done